tests/components/zha/test_discover.py

```diff
@@ -133,7 +129,7 @@ def _mock(
         assert entity_id is not None
         no_tail_id = NO_TAIL_ID.sub("", entity_map[key]["entity_id"])
         assert entity_id.startswith(no_tail_id)
-        assert set([ch.name for ch in channels]) == set(entity_map[key]["channels"])
+        assert {ch.name for ch in channels} == set(entity_map[key]["channels"])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5306536)